### DISCUSSION TITLE
How do I (re)start Rancher Desktop?

### DISCUSSION THREAD

**Discussion by huyz** [rancher-sandbox/rancher-desktop#2561] [Q&A]
(I've selected the containerd runtime)

When I run `nerdctl` it says:
```
Rancher Desktop is not running. Please start Rancher Desktop to use nerdctl
```

I launch the Rancher Desktop UI and then do what? I don't see a start, launch, or restart button anywhere.

**Comment by rak-phillip** [comment]
When you open Rancher Desktop, it will automatically run through some startup routines. You can see the progress bar in the lower left-hand side of the main window.

![image](https://user-images.githubusercontent.com/835961/180333914-caca316f-0c76-4f06-b65a-170018e16992.png)

If all goes well, you will see a blue Rancher Desktop logo in your tray menu when Rancher Desktop is running.

![image](https://user-images.githubusercontent.com/835961/180333185-698d8334-780f-4798-b91d-80707e7c8b0c.png)

This should be enough to start using nerdctl.

**Comment by huyz** [comment]
In my case, that wasn't happening. I don't know why. I think ended up having to do a factory reset to get it to work again.
